docs/configuration/backends/kubernetes.md

```diff
@@ -158,6 +158,7 @@ The following security annotations can be applied to the ingress object to add s
 | `ingress.kubernetes.io/public-key:VALUE`                 | Adds pinned HTST public key header.                                                                                                                                                                 |
 | `ingress.kubernetes.io/referrer-policy:VALUE`            | Adds referrer policy  header.                                                                                                                                                                       |
 | `ingress.kubernetes.io/is-development:false`             | This will cause the `AllowedHosts`, `SSLRedirect`, and `STSSeconds`/`STSIncludeSubdomains` options to be ignored during development.<br>When deploying to production, be sure to set this to false. |
+| `ingress.kubernetes.io/app-root:VALUE`             |  Explicitly make Ingress Controller redirects `/` requests to the defined path. |
```

Comment:
As we're documenting the Ingress controller, can we simplify this to

_Redirects all requests for `/` to the defined path._

and also have another sentence to clearly describe what happens for non-root paths:

_Non-root paths will not be affected by this annotation and handled normally._

Reply:
Make perfect sense. 👍